_posts/2022-06-22-community-meeting-financial-group-blog-report.md

```diff
@@ -0,0 +1,113 @@
+   * ADSA ([Academic Data Science Alliance](https://academicdatascience.org/)); sister org with overlapping member profile, have a mix of institutional, lab, and individual memberships a model.
+   * SORSE ([Society of Research Software Engineers](https://society-rse.org/)), [SSI (Software Sustainability Institute)](https://www.software.ac.uk/) Fellowship program, but different style of member model. SORSE aspires to represent RSEs globally.
+   * ACM HPC-SIG ([High Performance Computing Special Interest Group](https://dl.acm.org/sig/sighpc)). An example of being a part of a larger umbrella. 
+   * SIAM ([Society of Industrial and Applied Mathematics](https://www.siam.org)). Professional organization that holds traditional conferences and serves as a vehicle for professional, career affirming activities.
+   * [The Carpentries](https://carpentries.org/). Has a train-the-trainer and education model, with institutional memberships model.
+   * https://www.digitalinclusion.org/join/. Has a membership fee model primarily tied to strengthening a sense of community. It offers tiered services that distinguish between members and non-members. 
+
+
+### Question:
+
+*What are some examples of how we could use funding?*
+
+### Answers:
+
+* Key principles for allocating funding target priorities are
+   * Participatory budgeting. 
+Fund allocations and fund development priorities in US-RSE should be open to all to present perspectives and shape direction. 
+   * Practicality. 
+The scale, duration and likelihood of garnering financial support for an activity should also be considered in setting priorities.
+
+* The group identified some candidate near-term and longer-term possible priorities for broader consideration.
+   * Near term (1-2 years)
+      * Member management software that supports basic organization services and supports member communication and networking.
+      * Website support.
+      * Small grants for advocacy/messaging activities that raise awareness and grow US-RSE membership.
+      * Support for student interns to help with administrative work (e.g scheduling events, processing videos, enhancing newsletter).
+      * Supporting organization of local/regional groups.
+   * Longer term
+      * Executive Director / Community Manager / Development Officer [part time]
+      * Fellowships
+      * Development of training materials
+      * Development of certificate program
+      * Scholarships for people to earn the certificate
+      * Small prizes for contests to attract people into the RSE profession (e.g. K-12 outreach)
+
+
+### Question:
+*What are the next steps?*
+
+### Answers:
+
+* The next steps will be centered on the US-RSE Financial Sustainability Working Group/Standing Committee. ***We welcome anyone who is interested in this topic to please join!***
```

Comment:
I suggest adding a link here, and potentially at the beginning and/or end of the post to a form/Slack channel/email address so people can "click to join".